### PR TITLE
Misc: go fmt + copyright + Makefile fix + dependabot + test fix

### PR DESCRIPTION
**Pull request by lluiscampos**
* go fmt artifact, cli and utils packages
* [LICENSE] Update copyright year to 2021
* [Makefile] Fix gofmt test in extracheck target
It as missing a semi-colon, so on error it would echo "/bin/false".
Aligned a with the Makefile from mender-connect
* #314 
* [write_test] Adjust expected output after urfave/cli upgrade

### CODE REVIEW

**Comment by lluiscampos** [comment]
Goes with https://github.com/mendersoftware/meta-mender/pull/1262

Build @ [Pipeline 238991082](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/238991082)

**Comment by lluiscampos** [comment]
Build @ [Pipeline 238995576](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/238995576)

**Review by tranchitella — APPROVED** [review]

File: cli/mender-artifact/write_test.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func TestArtifactsWrite(t *testing.T) {
 	os.Args = []string{"mender-artifact", "write", "rootfs-image"}
 	err = run()
 	assert.Error(t, err)
-	assert.Contains(t, err.Error(), "Required flags \" n,  t,  f\" not set",
+	assert.Contains(t, err.Error(), "Required flags \"artifact-name, device-type, file\" not set",
```
Comment (oleorhagen) [inline cli/mender-artifact/write_test.go:40]: Finally!

File: Makefile [hunk 2]
```diff
@@ -105,9 +105,9 @@ test: tooldep
 
 extracheck:
 	echo "-- checking if code is gofmt'ed"
-	if [ -n "$$($(GOFMT) -d $(PKGFILES))" ]; then \
+	@if [ -n "$$($(GOFMT) -d $(PKGFILES))" ]; then \
+		"$$($(GOFMT) -d $(PKGFILES))" \
```
Comment (oleorhagen) [inline Makefile:109]: Why call the same command twice?

**Review by oleorhagen — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (lluiscampos) [reply Makefile:109]: On failures, print the diff to the user.

Same as we do for `mender` and `mender-connect` (and we were trying to do here but it was broken).

File: Makefile [hunk 3]
```diff
@@ -105,9 +105,9 @@ test: tooldep
...
 		echo "-- gofmt check failed"; \
-		echo $GOFMT -d $PKGFILES \
```
Comment (kacf) [inline Makefile:110]: Sneaky, well spotted!

**Review by kacf — APPROVED** [review]

**Pull request merged**